pkg/process/net/common_unsupported.go

```diff
@@ -15,6 +15,11 @@ func SetSystemProbePath(_ string) {
 	// no-op
 }
 
+// CheckPath is not supported
+func CheckPath() error {
+	return error.New("Not Supported")
```

Comment:
Can we return ebpf.ErrNotImpemented here as well for consistency ?

Reply:
Sure!